### DISCUSSION TITLE
Upgrade Laravel Version Support to 11

### DISCUSSION THREAD

**Discussion by MLF2kx** [apiato/apiato#764] [Ideas]
Laravel 11 is out there for about 4 months. I think it's a good idea to support it in Apiato and resolve conflicts because new projects doesn't tend to use older versions.

**Comment by Mohammad-Alavi** [comment]
Yea, I had this is mind for a long time now but unfortunately I couldn't find the time to do it yet.

**Reply by SDArtLoz** [reply]
Hello. Do you have plans when it could be possible? Thank you
